ampy/file_sync.py

```diff
@@ -155,15 +155,38 @@ def get_sync_info(pc_info, dev_info):
     # print(sync_info)
     return sync_info
 
-def file_sync_info(local_path, info_pathname, rtt_version):
+def should_exclude(name, excluded):
+  name = name.lower()
+  for exclusion in excluded:
```

Comment:
I think the code style is not the same as before, we need 4 spaces here to correspond with Python grammer lint.

Reply:
fixed